shopfloor_reception/services/reception.py

```diff
@@ -1406,7 +1406,7 @@ def set_quantity(self):
             },
             "quantity": {"type": "float"},
             "barcode": {"type": "string"},
-            "confirmation": {"type": "boolean"},
+            "confirmation": {"type": "string"},
```

Comment:
I think this should be now nullable
```suggestion
            "confirmation": {"type": "string", "required": False, "nullable": True},
```

Reply:
cc @jbaudoux

Reply:
I think so, yes.

Reply:
I don't think this was required, but did the change anyway.